README.md

```diff
@@ -170,12 +171,21 @@ Immediately triggers a new search for the given text
 
 - text `String` A word of sequence of words to search for
 
-#### `search.playResult(result)`
+**Returns:**
+- retVal `Promise` - A promise that will resolve with the results of the search you just performed.
+The promise will be rejected with a timeout error if the search takes too long.
+
+#### `search.playResult(searchText, result)`
 Immediately attempts to play the given result.  If we fail to play the given result
 an error will be thrown.
 
+- searchText `String` The search text that causes the search result to appear
```

Comment:
Quite possibly, but as a user I would still find it strange that I can't `playResult()` without first calling `performSearch()`.  There is no case where a user calling `playResult()` can get away with not calling `performSearch()` directly before it.

In it's previous state and your desired state `playResult()` will be a useless function by itself.  I don't see how crippling a function like that is a desired effect